client/app/app.routes.js

```diff
@@ -89,11 +89,10 @@
         },
         resolve: {
           checkUser: ['$http', checkUser],
-          mealsData: ['$http', '$stateParams', function($http, $stateParams) {
```

Comment:
I changed it back because it was causing problems and was fixed when I stopped injecting them - not sure why.